dcs-packaging-tool/dcs-packaging-tool-impl/src/test/java/org/dataconservancy/packaging/tool/impl/GeneralPackageDescriptionCreatorTest.java

```diff
@@ -370,6 +371,7 @@ public void simLinkCycleTest() throws Exception {
             Files.createSymbolicLink(link, subdir.toPath());
         } catch (UnsupportedOperationException e) {
             /* Nothing we can do if the system doesn't support symlinks */
+            FileUtils.deleteDirectory(tempDir);
```

Comment:
Just a note that with the use of the Junit TemporaryFolder rule you don't need to delete file and folders created under the temporary folder. It's done for you automatically so the FileUtils.deleteDirectory and calling delete on the directories is unnecessary.